SpaceDome/src/booster.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <memory>
+#include <AssimpLoader.h>
+#include "assimp/Importer.hpp"
+
+
+#include "mesh.h"
+#include "utility.h"
+#include "player.h"
+
+class Player;
+
+
+class Booster {
+
+    public:
+
+        Booster(){position = generateBoosterPos();}
+    
+        virtual void draw(const GLuint shaderBooster, glm::mat4 pMatrix, glm::mat4 vMatrix) const = 0;
+        virtual void deActivate(Player& player) = 0;
+        virtual void activate(Player& player) = 0;
+
+        void updateBooster();
+        glm::vec3 generateBoosterPos();
+        glm::vec3 getPosition() {return position;}
+        bool getIsDeactived(){ return isDeactived;}
```

Comment:
Vet inte om denna funktion behövs, men håll generellt funktionsnamnen så enkla och rättframma som möjligt:
```suggestion
        bool isActive() const { return ...;}
```